apertium-dan.dan.dix

```diff
@@ -60688,6 +60688,961 @@ TODO:
+<e lm="bilagt">                        <i>bilagt</i><par n="stolt__adj"/></e>
+<e lm="billedskøn">                    <i>billedskøn</i><par n="grøn__adj"/></e>
+<e lm="bindegal">                      <i>bindegal</i><par n="klar__adj"/></e>
+<e lm="binokular">                     <i>binokular</i><par n="klar__adj"/></e>
+<e lm="binomial">                      <i>binomial</i><par n="klar__adj"/></e>
+<e lm="bipolar">                       <i>bipolar</i><par n="klar__adj"/></e>
+<e lm="bitterfri">                     <i>bitterfri</i><par n="klar__adj"/></e>
+<e lm="bjergtagen">                    <i>bjergtag</i><par n="besked/en__adj"/></e>
+<e lm="bladet">                        <i>blade</i><par n="øve/t__adj"/></e>
+<e lm="blegnæbbet">                    <i>blegnæbbe</i><par n="øve/t__adj"/></e>
+<e lm="blokfri">                       <i>blokfri</i><par n="klar__adj"/></e>
+<e lm="blområdden">                    <i>blområd</i><par n="råd/den__adj"/></e>
+<e lm="blækket">                       <i>blække</i><par n="øve/t__adj"/></e>
+<e lm="blødsøden">                     <i>blødsød</i><par n="besked/en__adj"/></e>
+<e lm="bolchestribet">                 <i>bolchestribe</i><par n="øve/t__adj"/></e>
+<e lm="bolsjestribet">                 <i>bolsjestribe</i><par n="øve/t__adj"/></e>
+<e lm="bombesikker">                   <i>bombesik</i><par n="sik/ker__adj"/></e>
+<e lm="bondsk">                        <i>bondsk</i><par n="amerikansk__adj"/></e>
+<e lm="brandfri">                      <i>brandfri</i><par n="klar__adj"/></e>
+<e lm="brandgul">                      <i>brandgul</i><par n="klar__adj"/></e>
+<e lm="brandsikker">                   <i>brandsik</i><par n="sik/ker__adj"/></e>
+<e lm="brandskadet">                   <i>brandskade</i><par n="øve/t__adj"/></e>
+<e lm="bredbladet">                    <i>bredblade</i><par n="øve/t__adj"/></e>
+<e lm="bredfuld">                      <i>bredfuld</i><par n="klar__adj"/></e>
+<e lm="bredstribet">                   <i>bredstribe</i><par n="øve/t__adj"/></e>
+<e lm="bremsesikker">                  <i>bremsesik</i><par n="sik/ker__adj"/></e>
+<e lm="bronkial">                      <i>bronkial</i><par n="klar__adj"/></e>
+<e lm="brudfri">                       <i>brudfri</i><par n="klar__adj"/></e>
+<e lm="brudsikker">                    <i>brudsik</i><par n="sik/ker__adj"/></e>
+<e lm="brugsklar">                     <i>brugsklar</i><par n="klar__adj"/></e>
+<e lm="brunmasket">                    <i>brunmaske</i><par n="øve/t__adj"/></e>
+<e lm="brusket">                       <i>bruske</i><par n="øve/t__adj"/></e>
+<e lm="brødefuld">                     <i>brødefuld</i><par n="klar__adj"/></e>
+<e lm="brøstholden">                   <i>brøsthold</i><par n="besked/en__adj"/></e>
+<e lm="bunden">                        <i>bund</i><par n="besked/en__adj"/></e>
+<e lm="bundrådden">                    <i>bundrådd</i><par n="råd/den__adj"/></e>
```

Comment:
won't this get three d's in a row?

Reply:
Jo, det er en fejl. Lemmaet var duplikeret. Jeg ved ikke, hvordan det er sket.